Resources/Private/Partials/Backend/Application/Form/Rating.html

```diff
@@ -7,7 +7,13 @@
 <f:form.hidden property="subject" value="{subject}" />
 
 <div class="form-group">
-	<f:form.select name="application[{ratingField}]" options="{ratingOptions}" value="{ratingValue}" class="form-control" />
+	<f:form.select name="application[{ratingField}]" value="{ratingValue}" class="form-control" >
+		<f:for each="{ratingOptions}" as="option" key="optionKey">
```

Comment:
Please verify this is working on TYPO3v7. `<f:form.select/>` can only have content in TYPO3v8 and newer. In that case you should use `<f:form.select.option/>` BTW.